src/main/java/net/dv8tion/jda/api/events/message/react/GenericMessageReactionEvent.java

```diff
@@ -62,7 +62,7 @@ public GenericMessageReactionEvent(@Nonnull JDA api, long responseNumber, @Nulla
      *
      * @return The user id
      */
-    @Nonnull
+    @Nullable
     public String getUserId()
```

Comment:
This seems wrong to me

Reply:
Yep, it was a leftover change from when it could return null.